lib/cocoapods-core/source/manager.rb

```diff
@@ -234,6 +234,11 @@ def update_search_index_if_needed(changed_spec_paths)
       #                  A hash containing changed specification paths for each source.
       #
       def update_search_index_if_needed_in_background(changed_spec_paths)
+        if Gem.win_platform?
+          update_search_index_if_needed(changed_spec_paths)
+          exit
```

Comment:
why does this exit?

Reply:
Copy-paste bug - since it's one of the last things happening during the update, didn't catch it. Fixing and pushing with rebase.